src/main/java/core/basesyntax/service/serviceimpl/FruitReportCreateImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.serviceimpl;
+
+import core.basesyntax.exception.NegativeBalanceException;
+import core.basesyntax.service.interfaces.FruitReportCreate;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitReportCreateImpl implements FruitReportCreate {
+    private static final String SEMICOL = ",";
+
+    public String createReport(HashMap<String, Integer> fruitQuantity) {
+        StringBuilder builder = new StringBuilder();
+        builder.append("fruit").append(SEMICOL).append("quantity").append(System.lineSeparator());
```

Comment:
no need to append separate words, jsut have a while first row as a constant and put it in the constructor of the StringBuilder instead of appending